src/test/resources/authenticationTest/global_authentication_none.yml

```diff
@@ -0,0 +1,33 @@
+openapi: 3.0.1
+info:
+  description: Testing setting of global security parameter
+  title: Test
+  version: '0.0'
+paths:
+  /test:
+    get:
+      operationId: testPath
+      parameters:
+        - in: query
+          name: testDTO
+          required: true
+          schema:
+            $ref: '#/components/schemas/testDTO'
+      responses:
+        '200':
+          description: generateTestModel 200 response
+      tags:
+        - additional-models
```

Comment:
**question:** Is this required to have here for the tests? Otherwise, I'd suggest to remove it to keep the YAML files as focused as possible.

Reply:
well spottet